src/Main.java

```diff
@@ -70,8 +71,6 @@ public static void main(String[] args) {
         // 5. Instantiate Parser and parse input file to FloorInputEvents
         System.out.println("\n****** Generating System Input Events ******\n");
         Parser parser = new Parser();
-        //String inputFilename = "res/input-file.txt";
```

Comment:
For merge conflict resolution: 
- Save these changes somewhere
- Override this main with upstream -> becomes MainCombinedDMA
- Fix MainCombinedDMA to use this new code using input file name
- Also use input file name in MainFloorUDP

Reply:
Thanks! Yeah, realized after I should have just done this once you split the mains. I'll sort this, and I'll probably be making a few more (minor) changes to the config anyways to clean up them printfs. Nice job on the UDP splits!